### PR TITLE
Fixing a bunch of links

### PR DESCRIPTION
**Pull request by vturecek**
- Community links
- Adding notational conventions link to spec ToC
- Removing dead appendix link

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -70,9 +71,9 @@ One of the easiest ways to contribute is to participate in discussions. There ar
 
 | Item        | Value  |
 |---------------------|---|
-| Mailing List | Coming soon! |
-| Slack Channel       | Coming soon! |
-| Community meeting | Every other Tuesday at 10:30AM PST. Zoom meeting link: https://zoom.us/j/623691799?pwd=ZWc4SHFNdWpRUVVNYkdJWE9zVHpjZz09  |
-| Meeting Notes       | https://docs.google.com/document/d/1nqdFEyULekyksFHtFvgvFAYE-0AMHKoS3RMnaKsarjs/edit?usp=sharing |
+| Mailing List | https://groups.google.com/forum/#!forum/oam-dev |
+| Community meeting info | Bi-weekly (Starting October 23rd), Tuesdays 10:30AM PST  |
```
Comment (hongchaodeng) [inline README.md]:
```suggestion
| Community meeting info | Bi-weekly (Starting Oct 23, 2019), Tuesdays 10:30AM PST  |
```

Comment (hongchaodeng) [reply README.md]: Minor nit

Comment (vturecek) [reply README.md]: FIxed

**Comment by vturecek** [comment]
Done, fixed the date too (should be 22nd, not 23rd)

**Pull request merged**